.github/workflows/install-latest-snapshot-to-ubuntu-22-04.yml

```diff
@@ -1,5 +1,8 @@
 name: Install Latest Snapshot to Ubuntu 22.04
-on: [push]
+on:
```

Comment:
Let it run in every push because it also checks the server package and it just takes a couple of minutes.